apps/backend/src/routes/onboard/verification/+page.svelte

```diff
@@ -0,0 +1,83 @@
+<!-- JavaScript code -->
+<script lang="ts">
+  import { firestore } from '$lib/firebase';
+  import { authStore } from '$lib/stores';
+  import { doc, getDoc } from 'firebase/firestore';
+
+  // JavaScript code for Loading Button animation
+  let checkLoading = false;
+
+  // Get user UID
+  const uid = $authStore?.uid || 'defaultUid';
```

Comment:
Removed entirely because its not even being used anymore. It was suppose to be for pulling the data from firestore straight which has been replaced.